### DISCUSSION TITLE
Support for nunjucks ?

### DISCUSSION THREAD

**Discussion by NikhilNanjappa** [vercel/community#3433] [Help]
### Summary

My simple app uses the following notable technologies - webpack, express, nunjucks.

Repo link - https://github.com/NikhilNanjappa/bin-collection/tree/main

### Issue

When I deploy my app & visit the URL, I get an error: `Error: template not found: template.njk`. This is a template that im extending from within the node_modules from one of the npm packages. Its able to get the template properly on my local but not when deployed.

### My project settings

<img width="852" alt="Screenshot 2023-07-30 at 4 53 15 pm" src="https://github.com/vercel/community/assets/94963727/c0b59cf9-ad44-4bb4-80a0-9043cbdc6881">

My `vercel.json`
```json
{
  "version": 2,
  "name": "bin-collection",
  "builds": [
    { "src": "src/server/index.js", "use": "@vercel/node" }
  ],
  "routes": [
    { "src": "/(.*)", "dest": "src/server/index.js" }
  ]
}
```

### Example

https://bin-collect.vercel.app/

### Steps to Reproduce

https://bin-collect.vercel.app/_logs

**Comment by amyegan** [comment]
Hey @NikhilNanjappa. The guide about [using files in serverless functions](https://vercel.com/guides/how-can-i-use-files-in-serverless-functions) may be helpful for this. 

You may also want to take a look at the guide about [using Express with Vercel](https://vercel.com/guides/using-express-with-vercel). In particular, the [Drawbacks and Edge Cases](https://vercel.com/guides/using-express-with-vercel#drawbacks-and-edge-cases) and [Re-thinking our Application](https://vercel.com/guides/using-express-with-vercel#re-thinking-our-application) sections may be useful.

**Comment by NikhilNanjappa** [comment]
Hi @amyegan , thanks for getting back.

I’ve seen the links already. I’m not using files as mentioned in those examples so it doesn’t apply to me. I’m passing path of a folder (within node_modules) to nunjucks.configure and that’s it. So as long as node_modules is accessible, nunjucks library should do the rest.

Under drawbacks I only see that it says using templating engines are inefficient but it should still work?

It almost feels like node_modules is not accessible to my deployed app. Should I be making some configuration changes or am I looking in the wrong direction?
